addons/l10n_au/data/account.account.tag.csv

```diff
@@ -0,0 +1,10 @@
+id,name,applicability
+tag_au_tax_sale_10,Tag - au_tax_sale_10,taxes
```

Comment:
Does the name of the tags contains underscores? That is the value displayed to the user (but if legal tax name, that's fine for me)